src/comlink.ts

```diff
@@ -640,13 +651,12 @@ function requestResponseMessage(
   msg: Message,
   transfers?: Transferable[]
 ): Promise<WireValue> {
-  return new Promise((resolve) => {
+  return new Promise((resolve, reject) => {
     const id = Math.trunc(Math.random() * Number.MAX_SAFE_INTEGER).toString();
-    pendingListeners.set(id, resolve);
+    pendingListeners.set(id, { resolve, reject });
```

Comment:
Performance nit: should we use a tuple here to avoid v8 object creation? Or is that being too pedantic?